pytorch_toolkit/ote/ote/modules/arg_converters/mmdetection.py

```diff
@@ -128,7 +144,7 @@ def _get_extra_train_args(self, args):
 
         return classes_list_to_update_config_dict(args['config'], classes)
```

Comment:
`return self.get_extra_test_args(args)` or move it into separate function to exclude copy-paste?

Reply:
Evgeny, I copied the functions from the previous implementation of `MMDetectionCustomClassesArgConverter` made by Ilya earlier and I did not change them. I would prefer to move refactoring of the functions out of scope of this PR.

Please, note that for train and test the functions do a bit different things (different work with `classes_from_snapshot`, etc).

Reply:
Sorry, but my comment is mostly about a couple lines above, which are the same for `get_extra_train_args`, `get_extra_test_args`, `get_extra_compress_args` methods.

Reply:
No, they are a bit different -- different text of the messages, `logging.warning` vs `raise RuntimeError`, `get_extra_test_args` contains loading the `classes_from_snapshot` from `'.extra_params.yml'` file (whereas for train case it MUST NOT be done), etc

I agree that some common part may be extracted, but 
* different part of message texts will have to be passed to the common part
* I would like to move refactoring of the code outside the scope of this PR

Reply:
Ok, but not forget to add refactoring in the future to make the code great again.